app/controllers/api/v1/users_controller.rb

```diff
@@ -25,10 +25,11 @@ def update
       private
 
       def user_params
-        params.require(:user).permit(
+        params.permit(
```

Comment:
how are you passing the `:user` without the require?

Reply:
the front end is changing from passing straight up json with a top-level user key to sending a multipart form. This means I don't want or need to package the string params under a user key. The avatar, username, password, age all live at top level on the multipart form. Because nothing else is using this endpoint, we can make big changes like this but if we had dependencies on this I would have made this work in other ways.